gravitee-apim-portal-webui-next/src/app/api-details/api-tab-details/api-tab-details.component.scss

```diff
@@ -29,6 +29,23 @@
   &__tab-details-info {
     display: flex;
     flex-direction: column;
+    gap: 32px;
     grid-row: 1 / 2;
   }
 }
+
+.details-title {
```

Comment:
These two classes should probably be defined globally as those settings will be used in other areas... But we can do that after @audrey-tlm lets us know the different types of text for the new portal.

Reply:
Okay, so I'm deleting it for now